raco/myrial/scanner.py

```diff
@@ -9,11 +9,11 @@
             'ELSE', 'END', 'CONST', 'LOAD', 'DUMP', 'CSV', 'SCHEMA',
             'OPP', 'TIPSY', 'UDA', 'TRUE', 'FALSE', 'HASH', 'BROADCAST',
             'ROUND_ROBIN', 'UNTIL', 'CONVERGENCE', "SYNC", "ASYNC",
-            'ALTERNATE', 'PULL_IDB', 'PULL_EDB', 'BUILD_EDB']
+            'ALTERNATE', 'PULL_IDB', 'PULL_EDB', 'BUILD_EDB', 'ASC', 'DESC']
 
 types = ['INT', 'STRING', 'FLOAT', 'BOOLEAN', 'BLOB']
 
-comprehension_keywords = ['SELECT', 'AS', 'EMIT', 'FROM', 'WHERE']
+comprehension_keywords = ['SELECT', 'AS', 'EMIT', 'FROM', 'WHERE', 'ORDERBY']
```

Comment:
I think it's important to use the conventional syntax `ORDER BY`, even if it's a pain to deal with the embedded space in the token. Here's some advice on that: https://stackoverflow.com/questions/26596068/identifying-keyword-pairs-in-lex